onnxruntime/core/graph/model.cc

```diff
@@ -237,7 +237,7 @@ Status Model::Load(std::istream& model_istream, ModelProto* p_model_proto) {
   if (!p_model_proto) {
     return Status(ONNXRUNTIME, INVALID_ARGUMENT, "Null model_proto ptr.");
   }
-  google::protobuf::io::IstreamInputStream zero_copy_input(&model_istream);
+  google::protobuf::io::IstreamInputStream zero_copy_input(&model_istream, 1 << 20);
```

Comment:
If `1<<20` is a constant for multiple places, we'd better to define it as a `constexpr`.

Reply:
Thank you. I'll resolve it next week when I add the static_analysis job back.